public/js/controllers/viewBuildError.js

```diff
@@ -17,7 +23,19 @@ CDash.controller('BuildErrorController',
       $rootScope['title'] = cdash.title;
     }).finally(function() {
       $scope.loading = false;
+      $scope.setPage(1);
     });
+
+    $scope.setPage = function (pageNo) {
+      var begin = ((pageNo - 1) * $scope.pagination.numPerPage),
+          end = begin + $scope.pagination.numPerPage;
+
+      $scope.pagination.buildErrors = $scope.cdash.errors.slice(begin, end);
```

Comment:
I think this will run into the same off-by-one error fixed for viewTests in MR #374 

Note: slice (0, -1) extracts the first element through the second-to-last element in the sequence - which is not what I would have expected!